core/controllers/subscriptions.py

```diff
@@ -15,14 +15,15 @@
 """Controllers related to user subscriptions."""
 
 from core.controllers import base
+from core.domain import acl_decorators
 from core.domain import subscription_services
 from core.domain import user_services
 
 
 class SubscribeHandler(base.BaseHandler):
     """Handles operations relating to new subscriptions."""
 
-    @base.require_user
+    @acl_decorators.can_manage_subscriptions
```

Comment:
Consider `can_subscribe_to_users` instead. I'm thinking -- manage subscriptions to what, and manage whose subscriptions?